src/pages/Main/Ranking.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import RankList from "./RankList";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faChevronCircleLeft,
+  faChevronCircleRight,
+} from "@fortawesome/free-solid-svg-icons";
+import "./Main.scss";
+
+class Ranking extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      currentIndex: 0,
+    };
+  }
+
+  handleButtonClick = () => {
+    const { currentIndex } = this.state;
+    const imgWidth = 20;
+    const imgViewLength = 5;
+    const imgTranslate = imgWidth * imgViewLength;
+    if (currentIndex === 0) {
+      this.setState({
+        currentIndex: currentIndex - imgTranslate / 2,
+      });
+    } else {
+      this.setState({
+        currentIndex: currentIndex + imgTranslate / 2,
+      });
+    }
```

Comment:
```suggestion
    const { currentIndex } = this.state;
    this.setState({
      currentIndex: currentIndex === 0 ? currentIndex - 50 : currentIndex + 50,
    });
```
변수로 선언된 부분이 고정이라는 전제하에 위와 같이 간단하게 바꿀 수 있겠네요 :)

Reply:
0은 falsy한 값이라는 점을 이용한다면 
currentIndex === 0 을 !currentIndex로 더 줄여서 사용할 수도 있습니다.